code/modules/mob/living/say.dm

```diff
@@ -88,9 +90,12 @@ proc/get_radio_key_from_channel(var/channel)
 	if(GREY in mutations)
 		message = "<span class='grey'>[message]</span>"
 
-	else if(COMIC in mutations)
+	if(COMIC in mutations)

```

Comment:
Is there any sanitization or something that would break if it stacked multiple spans? Speaking in a foreign language on a radio channel with grey or comic already stacks 4 spans iirc.